File: PATTERNS/Pattern34/pattern34.py
n = int(input("Enter the row number:"))
for i in range(1, 2*n):
    for k in range(2*n - 1, i-1, -1):
        print("  ", end="")
    for j in range(1, i+1):
        if j == 1 or j == i:
            print("* ", end="")
        elif i == 2*n-1:
            print("* ", end="")
        else:
            print("  ", end="")
    print()

File: PATTERNS/Pattern34/Pattern34.go
package main

import "fmt"

func main() {
	var n int
	fmt.Printf("Enter the row number:")
	fmt.Scanln(&n)
	for i := 1; i <= 2*n-1; i++ {
		for k := 2*n - 1; k >= i; k-- {
			fmt.Print("  ")
		}
		for j := 1; j <= i; j++ {
			if j == 1 || j == i {
				fmt.Print("* ")
			} else if i == 2*n-1 {
				fmt.Print("* ")
			} else {
				fmt.Print("  ")
			}
		}
		fmt.Println()
	}

}
